CLI installation failure: expected `bool`, found integer

```
# cargo install undither --features=binary
    Updating crates.io index
  Installing undither v1.0.5
   Compiling autocfg v1.0.0
   Compiling libc v0.2.73
   Compiling bytemuck v1.3.1
   Compiling imgref v1.6.1
   Compiling either v1.5.3
   Compiling rgb v0.8.20
   Compiling loop9 v0.1.1
   Compiling itertools v0.9.0
   Compiling num-traits v0.2.12
   Compiling lodepng v3.0.0
   Compiling vpsearch v2.0.1
   Compiling undither v1.0.5
error[E0308]: mismatched types
  --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/undither-1.0.5/src/bin.rs:23:35
   |
23 |     state.decoder.color_convert = 0;
   |                                   ^ expected `bool`, found integer

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
error: failed to compile `undither v1.0.5`, intermediate artifacts can be found at `/tmp/cargo-installuYlADf`

Caused by:
  could not compile `undither`.

To learn more, run the command again with --verbose.

# rustc --version
rustc 1.44.0 (49cae5576 2020-06-01)
```

--- a/src/bin.rs
+++ b/src/bin.rs
@@ -20,7 +20,7 @@ https://github.com/kornelski/undither", args[0], env!("CARGO_PKG_VERSION"));
     }
 
     let mut state = State::new();
-    state.decoder.color_convert = 0;
+    state.decoder.color_convert = false;
     state.info_raw.colortype = ColorType::PALETTE;
     state.info_raw.set_bitdepth(8);
     let decoded = state.decode_file(&args[1]).unwrap();

